client/scraper/error_metadata.py

```diff
@@ -9,6 +9,14 @@
 
 class ErrorMetadata(Metadata):
 
+    def init_fields(self):
+        self.prop_map[FieldKeyword.STATUS] = None
+        self.prop_map[FieldKeyword.URL] = None
+        self.prop_map[FieldKeyword.REQUEST_URL] = None
+        self.prop_map[FieldKeyword.PROVIDER_URL] = None
+        self.prop_map[FieldKeyword.TITLE] = None
```

Comment:
I don't think we should have title in this. I find it misleading to have a title for a page that has errored out